src/SprykerEco/Zed/Payone/Business/Payment/MethodMapper/CashOnDelivery.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodMapper;
+
+use Generated\Shared\Transfer\OrderTransfer;
+use Generated\Shared\Transfer\PayoneAuthorizationTransfer;
+use Generated\Shared\Transfer\PayoneBankAccountCheckTransfer;
+use Orm\Zed\Payone\Persistence\SpyPaymentPayone;
+use Spryker\Shared\Kernel\Store;
+use SprykerEco\Shared\Payone\PayoneApiConstants;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\AbstractAuthorizationContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\PaymentMethod\CashOnDeliveryContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\PersonalContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\AuthorizationContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\BankAccountCheckContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\CaptureContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\DebitContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\PreAuthorizationContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\RefundContainer;
+use SprykerEco\Zed\Payone\Dependency\Facade\PayoneToGlossaryInterface;
+
+class CashOnDelivery extends AbstractMapper
```

Comment:
Shouldn't it be smth like `CashOnDeliveryMapper`?

Reply:
Please check other similar classes, they named accordingly, i just extenteded it with provided project files, in module level